toes/talk.py

```diff
@@ -35,7 +35,7 @@ def markov() -> str:
             words.append(word)
             word = choose_word(word)
                
-        return ' '.join(words)
+        return name.capitalize() + ": " + ' '.join(words)
```

Comment:
Could you do this outside of the `get_markov` function and put it in `add_talk_command` instead? This way `get_markov`  can continue returning a "pure" Markov chain output which can be used in other situations (e.g. the reply to pings).

Reply:
Also if you could use single quotes instead of double quote to match the style that'd be great.